executor/prepared.go

```diff
@@ -341,15 +341,18 @@ func ResetStmtCtx(ctx context.Context, s ast.StmtNode) {
 		sc.OverflowAsWarning = false
 		sc.TruncateAsWarning = !sessVars.StrictSQLMode || stmt.IgnoreErr
 		sc.InUpdateOrDeleteStmt = true
+		sc.DividedByZeroAsWarning = stmt.IgnoreErr
```

Comment:
should we consider StrictSQLMode here?

Reply:
According to [MySQL document](https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#ignore-strict-comparison)

> One conclusion to draw from the table is that when the IGNORE keyword and strict SQL mode are both in effect, IGNORE takes precedence. 

So we can ignore StrictSQLMode here.

Reply:
ok